docs/src/integrations/source/bitbucket.md

```diff
@@ -8,8 +8,8 @@ description: |
 
 It's possible to integrate a Platform.sh project with either the freely available Bitbucket Cloud product
 or with the self-hosted [Bitbucket Server](https://confluence.atlassian.com/bitbucketserver/).
-In both cases, you need to [install the Platform.sh CLI](/development/cli/_index.md#installation), if you haven't already done so,
-to set up the integration.
+In both cases, you need to [install the Platform.sh CLI](/development/cli/_index.md),
```

Comment:
Locally this one doesn't work for me. But [online it seems to be working](https://pr-2189-odiepqq-652soceglkw4u.eu-3.platformsh.site/integrations/source/bitbucket.html),

Reply:
It wasn't relative, which can be fixed.